squad/core/templates/squad/notification/diff.html.jinja2

```diff
@@ -81,6 +81,7 @@
     <h2>{{threshold_metric[0].name}}</h2>
     <ul>
       <li>Environment: {{threshold_metric[0].environment.slug}}</li>
+      <li>Metric name: {{threshold_metric[1].full_name}}</li>
```

Comment:
Why do you use both `0` and `1` indexes for the same data section? Seems odd. Maybe a comment?

Reply:
That's actually a tuple and the object gets built here: https://github.com/Linaro/squad/blob/1.43.1/squad/core/models.py#L1342

I scratched the back of my head exactly like you did and I don't like it how it looks, but I didn't want to refactor that atm.